Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -891,9 +720,6 @@ protected void RegisterResources()
 
             RegisterSingleInstance<ISearchEngine>(() => new SearchEngine(LoggerFactory, LibraryManager, UserManager));
 
-            CertificateInfo = GetCertificateInfo(true);
```

Comment:
what's the reason for dropping certificate here?

Reply:
So HTTPS is, currently, very broken, so it'll need a decent reimplementation to get it working again. The certificate loading code should be a a part of this reimplementation, in a seperate ICertificateProvider service, keeping it out of this monolithic file. I'll be working on moving all the registration code into another place next.

Reply:
Are you saying current HTTPS won't ever work, even if provided with some certificate?
If it works I'd rather leave it be for now and "move" it in subsequent PR rather than remove and add later (not that I personally use it, I trust nginx for HTTPS more, but a lot of users do use it).